packages/wallet/test/integration/cip30mapping.test.ts

```diff
@@ -49,6 +55,14 @@ describe('cip30', () => {
       ).not.toThrow();
     });
 
+    test('api.getCollateral', async () => {
```

Comment:
Missing cases:
- no utxo in the wallet
- combined utxo value in the wallet is less the amount
- combined utxo value more than amount and less then max_amount (more that 1 utxo needed to satisfy amount)
- amount more then max_amount
- amount is 0

Reply:
Added

Reply:
Isn't this being checked with this expect?: 
```
expect(() =>
  cslToCore.utxo(utxos!.map((utxo) => CSL.TransactionUnspentOutput.from_bytes(Buffer.from(utxo, 'hex'))))
).not.toThrow();
```

Reply:
I think by 'pure' @iadmytro means that they must have no assets (coin only)

Reply:
So, unspendables returned by the walled should be checkd? Shouldn't this be done when setting them?

Reply:
Alright. I've added this new validation